interface/src/app/mlmodel/selector.tpl.html

```diff
@@ -0,0 +1,13 @@
+<div class="well">
+  <h3>
+    Machine Learning Model
+  </h3>
```

Comment:
Aha - that makes sense. So maybe should I move these headers and styling to the ``gene-network.tpl.html`` template and just have the ``<select>`` element itself (with all its required logic) in this component template?